<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLabels</class>
 <widget class="QDialog" name="DialogLabels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Protocol Labels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Protocol Label Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ProtocolLabelTableView" name="tblViewProtoLabels">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>View Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbProtoView">
          <item>
           <property name="text">
            <string>Bits</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hex</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ASCII</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAdvancedSettings">
     <property name="title">
      <string>Advanced Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidgetAdvancedSettings"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConfirm">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProtocolLabelTableView</class>
   <extends>QTableView</extends>
   <header>urh.ui.views.ProtocolLabelTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
